packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -48,6 +55,15 @@ class DropdownList extends React.Component {
     this.searchString = '';
     this.pressed = false;
     this.listRef = null;
+    this.expanded = this.props.intl.formatMessage({ id: 'Terra.dropdownButton.expanded' });
+  }
+
+  componentDidMount() {
+    // Set focus to first focusable menu item
```

Comment:
```suggestion
    // Set focus to first focusable menu item. prevents focus from being set on button while navigating through dropdown items using keyboard in safari with Voice Over.
```

Reply:
Changes made. Thanks

Reply:
Can you provide me link to the commit in which above changes are made.

Reply:
I still see comment is not updated : https://github.com/cerner/terra-core/blob/e90e80c1489b61f12b36f8c626d30c6c0e9c68e4/packages/terra-dropdown-button/src/_DropdownList.jsx#L62

Reply:
Comments added as suggested.